content/blog/2019/mesh-federation/index.md

```diff
@@ -0,0 +1,1816 @@
+---
+title: Mesh Federation
+subtitle: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control
+description: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control.
+publishdate: 2019-08-12
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,multicluster,security,gateway,tls]
+---
+
+Managing applications across multiple Kubernetes clusters is currently one of the hottest topics in Kubernetes and Istio
+communities. You may need to split your applications between multiple clusters for different reasons, for example:
+
+-  scalability limits of a single cluster
+-  separation of production/dev/test/staging environments
+-  different security and compliance requirements for different parts of the application
+-  running parts of the application on multiple public clouds, and on a private cloud
+
+Currently there are [three patterns](/docs/setup/kubernetes/install/multicluster/) in Istio to create a
+[multicluster service mesh](/docs/concepts/multicluster-deployments/#multicluster-service-mesh).
+A _multicluster service mesh_ is a single _logical_ service mesh, spread among multiple Kubernetes clusters.
+
+Such a service mesh has the following characteristics:
+
+- **uniform identical naming** of namespaces and services. The `withdraw` service in the `accounts` namespace in one
```

Comment:
nit: do we need `identical` here? I think `uniform naming` is sufficient, no?

Reply:
Uniform vs identical: hostnames on the Web are uniform (FQDN) but not identical.
Uniform and identical naming means that the organisation defined the form of the names for all the clusters and also has means to enforce that the same name means the same thing in all its clusters.

Reply:
Actually, the Web is a wrong example. Let me remove the `indentical` and leave `uniform` naming, you are right.

Reply:
I'm not sure this distinction is entirely useful for a doc such as this. However, you could just say `identical naming` then, since identical would imply uniformity.